src/contexts/auth.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { createContext, Dispatch, useReducer } from 'react'
+
+import AuthReducer from 'reducers/auth'
+
+type AuthProvider = {
+  children: JSX.Element
+}
+
+const initialState = {}
```

Comment:
can we define how the `state` should be? i.e.

```
const initialState = {
  isAuthenticated: false,
}
```

Then the type of `state` will be fixed and won't need `any` as a type. And `any` is not really helpful.

Reply:
Fixed in e187091b02cc6415f847925c96333565b9580b51